bitrise.yml

```diff
@@ -1432,6 +1431,7 @@ workflows:
                 xcodebuild -showsdks | grep -E 'iphoneos|iphonesimulator'
                 exit 1
             fi
+    - deploy-to-bitrise-io@2.9.2: {}
```

Comment:
Moving the deploy step below the Build for testing to also have the xctestrun file in the Artifacts